qds_sdk/commands.py

```diff
@@ -1347,6 +1347,44 @@ def parse(cls, args):
         v["command_type"] = "DbTapQueryCommand"
         return v
 
+class JupyterNotebookCommand(Command):
+    usage = "jupyternotebookcmd <submit|run> [options]"
+
+    optparser = GentleOptionParser(usage=usage)
+    optparser.add_option("--path", dest="path", help="Path including name of the Jupyter notebook to be run with extension.")
```

Comment:
is the path a remote bucket path (S3/ Blob Storage etc) or local path?

```suggestion
    optparser.add_option("--path", dest="path", help="remote bucket path including name of the Jupyter notebook to be run with extension.")
```

Reply:
It is a file system path in Jupyter. Eg: "Users/tgavara@qubole.com/temp.ipynb"